### ISSUE TITLE
Can not run federated_main.py

### ISSUE THREAD

**Issue by zihao-ai** [PEILab-Federated-Learning/PromptFL#1]
Thanks for sharing your code. However, this repo still lacks necessary files to reproduce the results, such as sampling.py, draw.py, most of datasets.py. Can you provide these important files to help me run this project?

**Comment by rumaima** [comment]
Yes, can you please provide all the necessary files to run federated_main.py.

**Comment by Naiftt** [comment]
Yes please, can you provide these files?

**Comment by 06Liz** [comment]
Yes,can you provide the all files which can run?

**Comment by jione-park** [comment]
Yes,can you provide the all files which can run?

**Comment by FlashingPig** [comment]
Yes,can you provide the all files which can run?
